### DISCUSSION TITLE
how to get database instance

### DISCUSSION THREAD

**Discussion by joebnb** [sequelize/sequelize#15068] [Help & Questions]
## Issue Creation Checklist

[ x] I have read the [contribution guidelines](https://github.com/sequelize/sequelize/blob/main/CONTRIBUTING.md)

## Issue Description
how to get database instance  after  init
```js
 this.sequelize = new Sequelize('database', '', '', {
            dialect: 'sqlite',
            storage: this.config.DB_PATH,
            define: {
                freezeTableName: true,
                charset: 'utf8',
                collate: 'utf8_general_ci',
            },
            logging: false,
        });
const sqlite3 =  this.sequelize.getInstance();
sqlite3.config('something')  // i want do some run time configuration but can't  get the instance of database
```

**Comment by fzn0x** [comment]
What is the output did you expect from the database instance?

```js
// config.js
//  I'm curious what data you need from getDatabaseInstance().
module.exports = getDatabaseInstance().then(data => {
  ....,
  production: {
   // ...
    username: data.user,
  },
})
```

```js
// .sequelizerc
const path = require('path');

module.exports = {
  'config': path.resolve('config', 'config.js')
}
```

**Comment by joebnb** [comment]
> What is the output did you expect from the database instance?
> 
> ```js
> // config.js
> //  I'm curious what data you need from getDatabaseInstance().
> module.exports = getDatabaseInstance().then(data => {
>   ....,
>   production: {
>    // ...
>     username: data.user,
>   },
> })
> ```
> 
> ```js
> // .sequelizerc
> const path = require('path');
> 
> module.exports = {
>   'config': path.resolve('config', 'config.js')
> }
> ```

i want get sqlite3 instance which was instantiate by sequelize

**Comment by ephys** [comment]
You can try `sequelize.connectionManager.getConnection()`
